src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -1,7 +1,201 @@
 (ns status-im2.contexts.onboarding.profiles.view
```

Comment:
hmm profiles is a bit generic, perhaps switch-profile or login-page? or something like that maybe?